scripts/demo/demo_benchmark_app.sh

```diff
@@ -119,7 +119,7 @@ elif [[ $DISTRO == "ubuntu" ]]; then
     python_binary=python3
     pip_binary=pip3
 
-    system_ver=`cat /etc/lsb-release | grep -i "DISTRIB_RELEASE" | cut -d "=" -f2`
+    system_ver=$(grep -i "DISTRIB_RELEASE" -f /etc/lsb-release | cut -d "=" -f2)
```

Comment:
Shouldn't it be `grep -i "DISTRIB_RELEASE" /etc/lsb-release`? The `-f` option specifies a file to get _patterns_ from.